README.md

```diff
@@ -143,10 +151,10 @@ scheme_family:
   default: Build all schemes
 template:
   description:
-    - Set this to the name of a template to only build that one template instead of building all, which is the default
-    - Only building a single template is much faster then building all
+    - Set this to names of templates to only build those templates instead of building all, which is the default
```

Comment:
To document all possible usages, how about:

```
Set this to the name of a template or a list of template names to only build them instead of building all, which is the default
```

And ditto in the argument description in base16_builder.py

Reply:
Agreed, done.